app/Http/Controllers/Api/AssetsController.php

```diff
@@ -545,7 +545,8 @@ public function store(ImageUploadRequest $request)
         $asset->model_id                = $request->get('model_id');
         $asset->order_number            = $request->get('order_number');
         $asset->notes                   = $request->get('notes');
-        $asset->asset_tag               = $request->get('asset_tag', Asset::autoincrement_asset());
+        $asset->asset_tag               = $request->get('asset_tag', Asset::autoincrement_asset()); //yup, problem :/
+        // NO IT IS NOT!!! This is never firing; we SHOW the asset_tag you're going to get, so it *will* be filled in!
```

Comment:
I got confused by these comments. If I'm reading this right we can remove both comments right?

Reply:
I was hoping to record, for posterity, how, exactly this system is *actually* working, rather than "how we think it might work" so that the next time someone has to look at this problem, they don't go through the same mental gymnastics realizing how it _really_ works versus how we think it might want to work.

Very happy to workshop a better way of conveying this to future-us (who are going to look at this again in, like, a few months or years or something...)

Reply:
Ah gotcha. I was thinking these were debug comments for yourself as you worked through this PR. Keeping them there works for me 👍🏾